Index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mobile First Hussen Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="top-bar">
+        <div class="tool-bar">
+          <div class="logo-setting">
+            <div class="logo">
+              <a href="#"><img src="img/My-Logo.png" alt="HY" /></a>
+            </div>
+          </div>
+          <nav class="dash-setting">
+            <div class="icons">
+              <div class="icon-Union">
+                <img src="img/Union.png" alt="Three Dashs" />
+              </div>
```

Comment:
- [Optional] I see you have put a lot of effort into this @HussenYesufAlli, and it's looking good already. But as front-end developers, we need to be able to mimic a design into its identical website. I suggest you follow the suggestion from the previous reviewer and work on your design.
- [Optional] Please always look for `font-family`, `font-size`, `color`, `Alignment`, `spacing`, and other little details in a design and align your design toward it.
### Your design 
![image](https://user-images.githubusercontent.com/69077061/221372807-ea558ca9-f28b-4333-833d-48c09ffe6b53.png)

### Expected design.
<img width="197" alt="221351622-2b59406c-1d83-40a3-956e-033486a5293d" src="https://user-images.githubusercontent.com/69077061/221372815-0af79134-70ef-4208-acab-7e130f61a129.png">